src/robot/libraries/Process.py

```diff
@@ -439,11 +442,15 @@ def wait_for_process(self, handle=None, timeout=None, on_timeout='continue'):
         | ${result} =                 | Wait For Process | timeout=1min 30s | on_timeout=kill |
         | Process Should Be Stopped   |                  |                  |
         | Should Be Equal As Integers | ${result.rc}     | -9               |
+        | # Never-ending process      |                  |                  |
+        | ${result} =                 | Wait For Process | timeout=0        | on_timeout=kill |
+        | Process Should Be Stopped   | # Will return ``Fail``              |
```

Comment:
I don't think this is a useful example. It works exactly the same way as not using a timeout at all. I don't see why you'd ever use `timeout=0` or `timeout=NONE` in the first place and having such an example would just cause confusion. Something like `timeout=${timeout}` where the variable value may be `NONE` something else that disables the timeout could be useful, but such an example is too complicated here. In my opinion the earlier explanation about values that are ignored is enough.